app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -446,6 +451,11 @@ def repository_params
              ]
 
       keys += [{:docker_tags_whitelist => []}, :docker_upstream_name] if params[:action] == 'create' || @repository&.docker?
+      RepositoryTypeManager.repository_types.each do |_, type|
+        type.repository_remote_options.each do |option|
+          keys += [option.name] if params[:action] == 'create' || @repository&.generic?
```

Comment:
This worked for me:
```ruby
keys += [{option.name => []}] if params[:action] == 'create' || @repository&.generic?
```

Reply:
...but of course you'll need to edit based on the remote option type, so you have string and arrays